### PR TITLE
Bandmapfixup

### PR DESCRIPTION
**Pull request by airween**
Fix some small bugs in auto_cq mode:
* bandmap didn't refreshed, when Tlf worked in auto_cq mode
* cluster info didn't writed to dbfile (.bmdata.dat file) in every 10secs

### CODE REVIEW

File: src/autocq.c [hunk 1]
```diff
@@ -77,8 +84,15 @@ int auto_cq(void)
 		    stoptx();
 		    break;
 		}
+		gettimeofday(&timecnt, NULL);
```
Comment (dl1jbe) [inline src/autocq.c]: While correct, I think that is too complex. Just calling time_update() here does the whole job (same in line 105). It has the additional value that other time related work gets also done during auto_cq.

File: src/bandmap.c [hunk 2]
```diff
@@ -826,6 +817,16 @@ void bandmap_show() {
     move(cury, curx);			/* reset cursor */
 
     refreshp();
+
+    gettimeofday(&tv, NULL);
```
Comment (dl1jbe) [inline src/bandmap.c]: Maybe the bmdata_write_file should also be called from time_update(). Then we have concentrated all time dependent actions in one place. What do you think?

**Comment by dl1jbe** [comment]
Hi Ervin, in meantime I found time to fix the two problems in the master branch:

- aging and display during auto CQ and
- regular writing of the bandmap data 

Please check that it is working as expected. If so you can close the pull request. Thanks again for reporting the problem.

**Comment by airween** [comment]
Hi Thomas,

I've checked these two modifications, and looks like everything is good - thanks for the help.
I'll close this pull request.

**Pull request closed without merging**